_posts/research/subteams/2021-03-16-algorithmic-social-hri.md

```diff
@@ -46,19 +46,15 @@ traits of their partner.  In this stream of research, we explore how to imbue au
 </div>
 
 
-## 1.2 Reference Dependent Risk Attitudes in HRI
+## 1.2 Predictability in HRI
```

Comment:
Predictability is good , but "in HRI" is too generic. Pleae make it more compelling!

Reply:
reworked title